cmd/controller/controller.go

```diff
@@ -77,6 +77,11 @@ func AddConfigFlags(cmd *cobra.Command) {
 		"",
 		"Bind address to expose the pprof profiler (e.g. localhost:6060)",
 	)
+	cmd.Flags().Duration(
+		"startup-grace-period",
```

Comment:
Is it better to use a name more specific to image pull back off watcher? The current name might confuse reader with the start up graceful period of the controller itself (a.k.a the `initialDelaySeconds`). 

```suggestion
		"image-pull-backoff-watcher-startup-grace-period",
```

Reply:
Good point